types/route.d.ts

```diff
@@ -118,6 +118,7 @@ export type RouteHandlerMethod<
   RequestGeneric extends RequestGenericInterface = RequestGenericInterface,
   ContextConfig = ContextConfigDefault
 > = (
+  this: FastifyInstance<RawServer, RawRequest, RawReply>,
```

Comment:
i remember reading somewhere that this might be an indication of an antipattern. I'm looking into it now.

Reply:
okay nevermind this checks out as valid. LGTM :D